src/Mod/Sketcher/Gui/CommandSketcherTools.cpp

```diff
@@ -2404,6 +2405,39 @@ bool CmdSketcherRotate::isActive()
     return isCommandActive(getActiveGuiDocument(), true);
 }
 
+// Scale tool =====================================================================
+
+DEF_STD_CMD_A(CmdSketcherScale)
+
+CmdSketcherScale::CmdSketcherScale()
+    : Command("Sketcher_Scale")
+{
+    sAppModule = "Sketcher";
+    sGroup = "Sketcher";
+    sMenuText = QT_TR_NOOP("Scale transform");
+    sToolTipText = QT_TR_NOOP("Scale selected geometries. After selecting the center point you can either enter the scale factor, or select two reference points then scale factor = length(p2-center) / length(p1-center).");
+    sWhatsThis = "Sketcher_Scale";
+    sStatusTip = sToolTipText;
+    sPixmap = "Sketcher_Scale";
+    sAccel = "Z, P, S";
```

Comment:
I think we will have to do an exercise on the shortcuts when we merge all tools. 

Now Offset has "Z, T". Perhaps we could reserve Z, T for the "T"ransforms, so this could be Z, T, S. I am not a fan of 3 letter shortcuts though. It will require some thinking.

Reply:
Perhaps we can drop the Z and use T for transform? So just T, S.
Not sure T is available though.

Reply:
'T' is tangent constraint.

Originally, we had a limitation that 'T' would take away all the shortcuts starting with 'T', so no 'T, S'. Then tools would start with 'Z'. 

People using shortcuts have this hardwired in their minds, meaning that changing it is somewhere between "I will let you changing just once" and "Over my dead body".

Now, we have a timer based capturing, where 'T' and 'T, S' is possible. While we have it and it helps deconflict shorcuts, there is still the risk that a too slow 'T, S' causes a tangency to be executed, which is annoying.

Yet, with the introduction of new tools, there may be changes to current ones, so some now taken shortcuts may become free.

Then, even if 'T' and 'T, S' were ok, if we take 'T' for 'Transform', we should look which transforms we have before assigning letters. It is likely that we consider 'Symmetry' a transform, it would be odd to have one "transform" as "Z, S" and another at 'T, S'.

That is why I think it makes sense to delay the exercise.

With this in mind, an option would be not to assign a shortcut to this tool, or any other tool we have introduced lately, so that users do not get used to a shortcut that we know we will likely change.